de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/runner/RunnerNodeModel.java

```diff
@@ -401,8 +401,7 @@ private void applyJoinCommandToSimulation(CombinedFskPortObject fskObj,
         if (originalOutputParameters.containsKey(joinRelation.getSourceParam())) {
           // replace join command with unique identifier of parameter
           String command_new = joinRelation.getCommand().replaceAll("\\[([^<]*)\\]",
-              originalOutputParameters.get(joinRelation.getSourceParam()));
-          String target = "";
+              joinRelation.getSourceParam());
```

Comment:
@AhmadrSwaid : Is there a reason you don't use the originalOutputParameters here anymore or was it not working?

Reply:
Yes, the JoinRelation holds the global unique parameter Id and I used the originalOutputParameters as indicator to which relation are relevant to the current joined part.